server/modules/marketo/service.js

```diff
@@ -33,6 +33,13 @@ const parseResult = ({ result = [] } = {}) => {
 		throw error;
 	}
 
+	if (res.status === 'timeout') {
+		const error = new Error('Marketo timeout');
+		error.type = constants.API_TIMEOUT_ERROR;
+		error.reason = res.reasons;
+		throw error;
```

Comment:
Do we want to fail silently?

Reply:
As in don't return an error to the user if Marketo fails?

Reply:
Ah sorry, ignore me I thought the marketo was a secondary thing rather than the main reason!